apollo-sqlite-cache/src/main/sqldelight/com/apollographql/apollo/cache/normalized/sql/cache.sq

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE records (
+  _id INTEGER PRIMARY KEY AUTOINCREMENT,
+  key TEXT NOT NULL,
+  record TEXT NOT NULL
+);
+
+CREATE INDEX idx_records_key ON records(key);
+
+recordForKey:
+SELECT key, record FROM records WHERE key=?;
+
+insert:
+INSERT INTO records (key, record) VALUES (?,?);
+
+update:
+UPDATE records SET key=:key, record=:record WHERE key=:key;
```

Comment:
Nitpicking, maybe not update key ? 

```suggestion
UPDATE records SET record=:record WHERE key=:key;
```

Reply:
I saw this too but wanted to keep the same as before 😊 This was like this before.